### PR TITLE
[2.5] Update flower CLI

### PR DESCRIPTION
**Pull request by yanchengnv**
Fixes # .

### Description

This PR is to use the CLI of the latest Flower release.

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Quick tests passed locally by running `./runtest.sh`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated.

### CODE REVIEW

File: examples/hello-world/hello-flower/jobs/hello-flwr-pt/app/custom/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+clientapp = "flwr_pt.client:app"
+
+[tool.flwr.app.config]
+num-server-rounds = 3
+
+[tool.flwr.federations]
+default = "local-simulation"
+
+[tool.flwr.federations.local-simulation]
+options.num-supernodes = 2
```
Comment (YuanTingHsieh) [inline examples/hello-world/hello-flower/jobs/hello-flwr-pt/app/custom/pyproject.toml:34]: we might need to add in the documentation (or have links to flower documentation) to explain how these mean or what things can user configure in this "pyproject.toml"

Comment (holgerroth) [reply examples/hello-world/hello-flower/jobs/hello-flwr-pt/app/custom/pyproject.toml:34]: Yes, will be added once flower has the docs.

**Review by holgerroth — APPROVED** [review]
Works well on my side.

**Pull request merged**